hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -1796,6 +1866,34 @@ private void verifyChunksLength(ContainerProtos.ChunkInfo peerChunkInfo, Contain
     }
   }
 
+  /**
+   * If we do not have the previous chunk for the current entry, abort the reconciliation here. Currently we do
+   * not support repairing around holes in a block, the missing chunk must be obtained first.
+   */
+  private boolean previousChunkPresent(BlockID blockID, long chunkOffset,
+                                       NavigableMap<Long, ContainerProtos.ChunkInfo> localOffset2Chunk) {
+    long localID = blockID.getLocalID();
+    long containerID = blockID.getContainerID();
+    if (chunkOffset != 0) {
```

Comment:
We can do this in reverse ans reduce one level of indentation
```suggestion
    if (chunkOffset == 0) {
        return true;
    }
```